index.html

```diff
@@ -13,7 +13,11 @@
 </head>
 
 <body>
-
+  <div class="pai">
+  <button class="estilo" name="button"> <i class="fas fa-plus"></i>Add to Playlist </button>
```

Comment:
Não é necessário o name. 
```suggestion
  <button class="estilo"> <i class="fas fa-plus"></i>Add to Playlist </button>
```